packages/babel-preset-mapbox/index.js

```diff
@@ -1,16 +1,33 @@
 'use strict';
 const isPlainObj = require('is-plain-obj');
 
-module.exports = function preset(context, opts) {
+module.exports = function(api, opts) {
+  const { plugins, presets } = preset.call(this, api, opts);
+
+  // this is done in order to easily test this module
+  const requireModule = entity =>
+    Array.isArray(entity)
+      ? [require(entity[0]).default, entity[1]]
```

Comment:
Are all of them going to have `default` exports? None might be CommonJS?

Reply:
I am afraid yes, that what I found in `cra `and `next`  babel preset. This might be a problem if we use some third party (non-babel) preset which doesn't export default. You think its worth looking into this?

Reply:
Might be worth creating a little wrapper function that gets `default` only if it exists.